### PR TITLE
Add a param converter + route enhancers for entity revisions

### PR DESCRIPTION
**Pull request by dawehner**

### CODE REVIEW

**Comment by Berdir** [comment]
Can we put revision into the title somehow? was very confused when reading that ;)

Probably just "entity types" => "entity revisions"?

Also overlaps with the (old) info-yml PR, so should be rebased when that landed.

**Comment by dawehner** [comment]
Ha, good point with the title :)

> Also overlaps with the (old) info-yml PR, so should be rebased when that landed.
> Well, yeah I just merged the info.yml one is, as otherwise phpunit doesn't work.

**Comment by dawehner** [comment]
Made a quick rebase

File: src/ParamConverter/EntityRevisionParamConverter.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+use Drupal\Core\Entity\EntityManagerInterface;
+use Drupal\Core\ParamConverter\ParamConverterInterface;
+use Symfony\Component\Routing\Route;
+
+/**
+ * Parameter converter for single revisions.
+ */
+class EntityRevisionParamConverter implements ParamConverterInterface {
+
+  /** @var \Drupal\Core\Entity\EntityManagerInterface */
```
Comment (fago) [inline src/ParamConverter/EntityRevisionParamConverter.php]: Let's follow core coding style here?

File: tests/Unit/ParamConverter/EntityRevisionParamConverterTest.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\entity\ParamConverter\EntityRevisionParamConverter;
+use Symfony\Component\Routing\Route;
+
+/**
+ * @coversDefaultClass \Drupal\entity\ParamConverter\EntityRevisionParamConverter
+ * @group entity
+ */
+class EntityRevisionParamConverterTest extends \PHPUnit_Framework_TestCase {
+
```
Comment (bojanz) [inline tests/Unit/ParamConverter/EntityRevisionParamConverterTest.php:22]: Missing comment.

**Comment by dawehner** [comment]
There we go

**Comment by dawehner** [comment]
Should we merge this one in?

**Comment by bojanz** [comment]
The history got completely messed up here somehow, the labels don't match the actual commits.

Squashed and merged in 787c3c828d96fdfb9c072a98da8b6a56f4a89718.

**Comment by dawehner** [comment]
Thank you for the merge + cleanup!

**Pull request closed without merging**